tools/scripts/wasm.toml

```diff
@@ -383,11 +391,8 @@ script = '''
 exit_on_error true
 cd ffi/diplomat
 
-# Duckscript doesn't support globs in rm, so we delete the relevant files
-rm ./wasm/lib/api.mjs
-rm ./wasm/lib/diplomat-runtime.mjs
-rm -r ./wasm/docs/
-mkdir ./wasm/docs
+rm -r ./wasm/lib ./wasm/docs
+mkdir ./wasm/lib ./wasm/docs
```

Comment:
Suggestion: Split this into multiple lines because duckscript `mkdir` only wants one argument

https://github.com/sagiegurari/duckscript/blob/master/docs/sdk.md#std__fs__CreateDirectory